File: Crash Override.py
def alias_gen(f_name, l_name):
    f_g = FIRST_NAME.get(f_name[0].upper(),0)
    l_g = SURNAME.get(l_name[0].upper(),0)
    return f_g+" "+l_g  if f_g!=0 and l_g!=0 else 'Your name must start with a letter from A - Z.'

File: Crash Override/Crash Override.js
function aliasGen(a,b){
    if(a[0].match(/^[a-z]/i) && b[0].match(/^[a-z]/i)){
      return firstName[a[0].toUpperCase()]+" "+surname[b[0].toUpperCase()]
    }else{
      return "Your name must start with a letter from A - Z."
    }
}
